beacon-chain/sync/rpc_blob_sidecars_by_root.go

```diff
@@ -80,7 +80,7 @@ func (s *Service) blobSidecarByRootRPCHandler(ctx context.Context, msg interface
 		sc, err := s.cfg.blobStorage.Get(root, idx)
 		if err != nil {
 			if db.IsNotFound(err) {
-				log.WithError(err).Debugf("BlobSidecar not found in db, root=%x, index=%d", root, idx)
+				log.WithError(err).Debugf("Peer requested blob sidecar by root which was not found in db, root=%x, index=%d", root, idx)
```

Comment:
Suggestion: This would be better to remove the templating and use fields. Easier to collect and measure this log